workers/unity/Assets/Fps/Scripts/SetupLogic/AndroidWorkerConnector.cs

```diff
@@ -37,22 +37,30 @@ public class AndroidWorkerConnector : MobileWorkerConnector, ITileProvider
 
         public string IpAddress { get; set; }
 
-        public async void TryConnect()
-        {
-            await Connect(WorkerUtils.AndroidClient, new ForwardingDispatcher()).ConfigureAwait(false);
-        }
-
         private void Awake()
         {
             connectionController = GetComponent<ConnectionController>();
+
+            if (!ShouldConnectLocally)
+            {
+                var textAsset = Resources.Load<TextAsset>("DevAuthToken");
+                if (textAsset != null)
+                {
+                    DevelopmentAuthToken = textAsset.text.Trim();
+                }
+                else
+                {
+                    Debug.LogWarning("Unable to find DevAuthToken.txt in the Resources folder.");
```

Comment:
Do we plan to add tooling to generate these easily? Currently a user has to invoke:

```
spatial project auth dev-auth-token create --description "My token" | jq <some_parsing_magic_here> > workers/unity/Assets/Resources/DevAuthToken.txt
```

Reply:
I would like to add some kind of tooling for this. maybe a simple menu button for now that will just generate a token that is valid for a few days.

Reply:
Should be a quick one to build as well?